server/tinylicious/src/services/dbFactory.ts

```diff
@@ -4,10 +4,16 @@
  */
 
 import { IDb, IDbFactory } from "@fluidframework/server-services-core";
+import { Provider } from "nconf";
 import { InMemoryDb } from "./inMemorydb";
+import { LevelDb } from "./levelDb";
 
 export class DbFactory implements IDbFactory {
-    private readonly db = new InMemoryDb();
+    private readonly db;
+
+    constructor(config: Provider) {
+        this.db = config.get("db:inMemory") ? new InMemoryDb() : new LevelDb(config.get("db:path"));
```

Comment:
What will the default be? For back compat reasons with our existing usages the default should be inMemory

Reply:
Good point! Even though the config says inMemory: true, we will want to flip it have levelDb as an option. In every other case, use the inMemoryDB.

@kurtb: Can you do it?

Reply:
Default config is inMemory